launcher/src/store/services.js

```diff
@@ -302,15 +302,15 @@ export const useServices = defineStore("services", {
         },
         {
           id: 9,
-          name: "ssv network",
-          service: "BloxSSVService",
+          name: "SSV Network",
```

Comment:
If the `name` is a label on the UI I'd suggest using their `ssv.network` name, like on their website: https://ssv.network/